src/libs/ValidationUtils.ts

```diff
@@ -297,6 +297,13 @@ function isValidLegalName(name: string): boolean {
     return CONST.REGEX.ALPHABETIC_AND_LATIN_CHARS.test(name);
 }
 
+/**
+ * Checks that the provided name doesn't contain special characters or numbers
+ */
+function isValidPersonName(value: string) {
+    return /^[^\d^!#$%*=<>;{}"]+$/.test(value);
+}
```

Comment:
Hi folks, this RegExp allows symbols like "?" or "@" – did you consider using the `CONST.REGEX.ALPHABETIC_AND_LATIN_CHARS` here, or do we allow non-latin symbols in the names?

Reply:
Yes, just found it, thanks 👍